qemu/tests/uefi_pkg.py

```diff
@@ -43,14 +45,17 @@ def check_element_filename(filename, file_list):
         err_str += "invalid file. The invalid file is '%s'" % filename
         test.assertIn(filename, file_list, err_str)
 
-    def check_element_mode(mode):
+    def check_element_mode(mode, mode_value):
```

Comment:
suggest mode_value -> expected_mode or expected, then it's clear that which is expected, and which is tested, or line57: err_str = "The expected mode is '%s' instead of '%s'", the former may be better.

Reply:
Also for commit header, suggest to add the py name, e.g. 
uefi_pkg: xxx

Reply:
Thanks Zhenchao, the updates you mentioned are clearer. Many thanks.